arch/arm64/boot/dts/rockchip/rk3588s-rock-5a.dts

```diff
@@ -28,7 +28,7 @@
 	fan0: pwm-fan {
 		compatible = "pwm-fan";
 		#cooling-cells = <2>;
-		cooling-levels = <0 64 128 192 255>;
+		cooling-levels = <100 160 190 200 215 235 255>;
```

Comment:
> My fan from radxa starts spinning from pwm value 135, does your fan starts at value 100?
I have replaced the fan with a new one, and it will only start running at around 160. However, after the temperature drops to 100, the official fan should still be able to rotate